src/components/GloballySearched.js

```diff
@@ -147,7 +157,7 @@ function Song({ song, handleSelect }) {
       onClick={songClickRedirect}
     >
       <Typography variant='p' sx={styles.carouselSongTitle}>
-        {song.name}
+        {song.id}
```

Comment:
i think this should stay as .name? otherwise it becomes this! 

<img width="997" alt="Screenshot 2023-06-21 at 4 32 09 PM" src="https://github.com/KDhieb/cpsc-455-project/assets/43652006/0ce478e5-d6e0-4f21-b38b-17b787fa88f2">